packages/server/graphql/intranetSchema/mutations/checkRethinkPgEquality.ts

```diff
@@ -44,7 +45,15 @@ const checkRethinkPgEquality = {
       description: 'Whether the output should be written to file'
     }
   },
-  resolve: async (_source: unknown, {tableName, maxErrors, writeToFile}, {authToken}) => {
+  resolve: async (
+    _source: unknown,
+    {
+      tableName,
+      maxErrors,
+      writeToFile
+    }: {tableName: string; maxErrors?: number; writeToFile?: boolean},
```

Comment:
+1 optional GraphQL types should be `maxErrors: number | null | undefined` since `null` will be passed when omitted